src/views/Settings/Tables.tsx

```diff
@@ -1,5 +1,23 @@
-import { Heading } from '@chakra-ui/react';
-
-const Tables = () => <Heading>MOCK: Stoliki</Heading>;
+import { VStack, Box, Flex } from '@chakra-ui/react';
+import RoomTitle from '../../components/PlanRoom/RoomTitle';
+import SaveBoard from '../../components/PlanRoom/SaveBoard';
+import Board from '../../components/PlanRoom/Board';
 
+const Tables = () => (
+  <Box w="100%" minWidth="30em" h="36em">
+    <Flex w="100%" p="1em">
+      <VStack w="100%" spacing={3} align="stretch">
```

Comment:
We should use consistent declarations using quotes instead of braces, so you should change spacing from `{3}` to `"3"`.

Reply:
you're right